lib/passageidentity/auth.rb

```diff
@@ -20,7 +20,11 @@ def fetch_app()
         return response.body["app"]
       rescue Faraday::Error => e
         raise PassageError,
-              "failed to get Passage App. Http Status: #{e.response[:status]}. Response: #{e.response[:body]["error"]}"
+              PassageError.new(
```

Comment:
I can't say I'm very familiar with the error syntax in Ruby, but [this rubocop rule](https://github.com/rubocop/ruby-style-guide#exception-class-messages) seems to suggest that the arguments are either an exception instance, or an exception class and a string message. I'm not sure about an instance as the message.

Reply:
Ok that makes sense.  I can make that update